card-gateway/Model/CardPaymentDataFormatter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+declare(strict_types=1);
+
+namespace Pronko\LiqPayCardGateway\Model;
```

Comment:
Please move this class to the Pronko\LiqPayCardGateway\Gateway\Request directory.